image_resize.py

```diff
@@ -0,0 +1,20 @@
+from PIL import Image
+import os
+
+folder = "assets/images"
+new_size = (800, 600)
```

Comment:
Does this force all images to be 800 x 600?

Is there a way to crop these to the correct ratio before resizing. My worry is that images would be warped or distorted.